bld/namelist_files/namelist_definition.xml

```diff
@@ -2955,12 +2955,31 @@ Default: 10
 <!-- Moist Convection and Microphysics -->
 
 <entry id="deep_scheme" type="char*16" category="conv"
-       group="phys_ctl_nl" valid_values="ZM,UNICON,off,CLUBB_SGS" >
+       group="phys_ctl_nl" valid_values="ZM,UNICON,off,CLUBB_SGS,YOG" >
 Type of deep convection scheme employed.  'ZM' for Zhang-McFarlane;
-'off' for none; or 'UNICON' which doesn't distinquish shallow and deep.
+'off' for none;'YOG' for Yanni-O'Gorman; or 'UNICON' which doesn't
```

Comment:
Yuval-O'Gorman

Reply:
Done with @mondus' suggestion below in https://github.com/m2lines/CAM-ML/pull/7/commits/6c9c99c496d65afaf0faf13ad0314ea5f3a6d387